active-directory-new-domain/azuredeploy.json

```diff
@@ -159,24 +167,20 @@
     "adBEAddressPoolID": "[concat(variables('adlbID'),'/backendAddressPools/',variables('adLBBE'))]",
     "adDataDisk": "ADDataDisk",
     "adDataDiskSize": 1000,
-    "vnetTemplateUri": "[concat(parameters('assetLocation'),'vnet.json')]",
-    "vnetwithDNSTemplateUri": "[concat(parameters('assetLocation'),'vnet-with-dns-server.json')]",
-    "adModulesURL": "[concat(parameters('assetLocation'),'CreateADPDC.ps1.zip')]",
-    "adConfigurationFunction": "CreateADPDC.ps1\\CreateADPDC",
-    "apiVersion": "2015-06-15"
+    "adConfigurationFunction": "CreateADPDC.ps1\\CreateADPDC"
   },
   "resources": [
     {
       "type": "Microsoft.Storage/storageAccounts",
       "name": "[variables('storageAccountName')]",
-      "apiVersion": "[variables('apiVersion')]",
+      "apiVersion": "2015-06-15",
```

Comment:
Update Microsoft.Storage apiVersion to "2016-01-01" and replace its properties with:
      "sku": {
        "name": "Standard_LRS"
      },
      "kind": "Storage",